queue_processors/autoreduction_processor/post_process_admin.py

```diff
@@ -177,6 +177,28 @@ def _load_reduction_script(self, instrument_name):
         """ Returns the path of the reduction script for an instrument. """
         return os.path.join(self._reduction_script_location(instrument_name), 'reduce.py')
 
+
+    def specify_instrument_directories(self,
+                                       instrument_output_directory,
+                                       excitations,
+                                       temporary_directory):
+        """
+        Specifies instrument directories, including removal of run_number folder
+        if excitations instrument
+        :param instrument_output_directory: (str) Ceph directory using instrument, proposal, run no
+        :param excitations: (list) List of excitations instruments
```

Comment:
Since the list of excitation instruments isn't likely to change could we either:

- (Preferred) Change the param to `remove_run_no_dir` then make the caller do the check, this also means if other instruments want the same we don't have to touch this code later
- Alternatively, store the list of excitation instruments in the class init, since we already store the current instrument too so it doesn't get passed in